model/clusters_mgmt/v1/add_on_type.model

```diff
@@ -36,4 +36,10 @@ class AddOn {
 
 	// Used to determine how many units of quota an add-on consumes per resource name.
 	ResourceCost Float
+
+	// The namespace name in which the addon is installed.
+    TargetNamespace String
+
+    // The mode in which the addon is installed.
+    InstallMode String
```

Comment:
What are the possible values of this install mode and what do they mean? Consider adding a enumerated type for that:

```
enum AddOnInstallMode {
        // This mode means blah, blah, blah ...
        FirstMode

        // This mode means ....
        SecondMode
}
```

That should go in a separate `add_on_install_mode_type.model` file.

Reply:
Also please remember to indent the code only with tabs.

Reply:
This should be an enum. Currently accepted values should be `AllNamespaces` and `SingleNamespace`.